playground/frontend/lib/modules/examples/components/example_list/expansion_panel_item.dart

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:playground/constants/sizes.dart';
+import 'package:playground/modules/examples/models/example_model.dart';
+import 'package:playground/pages/playground/states/playground_state.dart';
+import 'package:provider/provider.dart';
+
+class ExpansionPanelItem extends StatelessWidget {
+  final ExampleModel example;
+
+  const ExpansionPanelItem({Key? key, required this.example}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Consumer<PlaygroundState>(
+      builder: (context, state, child) => GestureDetector(
+        onTap: () {
+          if (state.selectedExample != example) {
+            state.setExample(example);
+            state.setExamplesTitle(example.name);
```

Comment:
Do we need an example title on the state? Maybe we can just create getter inside title which will return it.